### PR TITLE
Gateway integration

### LINKED ISSUE #211: change "logging in with gitlab"
Could it just say "logging in" or maybe not even anything at all?

### PR DESCRIPTION
**Pull request by ableuler**
Merge after https://github.com/SwissDataScienceCenter/renku-gateway/pull/20
Merge together with https://github.com/SwissDataScienceCenter/renku/pull/205
closes #211, #196, #127

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -23,11 +23,11 @@ configPromise.then((res) => {
     const store = createStore(reducer);
 
     const client = new GitlabClient(
-      params.GITLAB_URL + '/api/v4/',
-      cookies.get('gitlab_token'),
+      params.GATEWAY_URL + '/v4/',
```
Comment (rokroskar) [inline src/index.js]: this is a gitlab-specific API version -- why is this appended directly to the `GATEWAY_URL`?

Comment (ableuler) [reply src/index.js]: I agree, the decision which gitlab api version to use should be delegated from the client to the gateway configuration in order to be more future-proof. I simply didn't bother changing this yet.

Comment (rokroskar) [reply src/index.js]: would be good to do https://github.com/SwissDataScienceCenter/renku-gateway/issues/9 asap then this is a non-issue

Comment (rokroskar) [reply src/index.js]: yes, that's what I meant. You may specify an api version - the default is `v4`.

Comment (ableuler) [reply src/index.js]: 👍

**Comment by ableuler** [comment]
@rokroskar I changed the corresponding PR on the Renku repo.

**Pull request merged**